cypress/integration/RichTextEditor.spec.ts

```diff
@@ -7,8 +7,8 @@ import {
 } from '../../packages/rich-text/src/helpers/nodeFactory';
 
 function expectRichTextFieldValue(expectedValue, editorEvents?) {
-  cy.getRichTextField().then((field) => {
-    expect(field.getValue()).to.deep.eq(expectedValue);
+  cy.getRichTextField().should((field) => {
+    expect(field.getValue()).to.deep.equal(expectedValue);
```

Comment:
The key to the removal of most `cy.wait()` calls. We still have some flaky tests (e.g. hyper links)